frontend/src/views/camp/Dashboard.vue

```diff
@@ -404,6 +404,11 @@ export default {
 
     const { categories, periods, collaborators, progressLabels } =
       await loadAndProcessCollections(this.camp())
+    const periodHrefs = await this.camp()
+      .periods()
+      .items.map((period) => period._meta.self)
+    await this.api.get().days({ periods: periodHrefs })
```

Comment:
I like the idea of this!
However, right now no filter is applied to the request:
![Screenshot from 2023-11-20 14-03-47](https://github.com/ecamp/ecamp3/assets/7566995/912f4cd9-932d-4a55-a190-0fc9a35b104b)
For one, that might be because the filter in the API is currently called `period` / `period[]`, not `periods` (https://pr4135.ecamp3.ch/api/#operations-tag-Day).
Secondly, hal-json-vuex currently does not support array parameters the way that API Platform / PHP expects them. If you pass an array of multiple periodHrefs, this will just get stringified to a comma separated list of URIs: `/days?period=/periods/1a2b3c4d,/periods/0f9e8d7c`
We should really start supporting these kinds of filters, but it will need to live behind a flag in hal-json-vuex, because the `period[]` parameter form isn't really standardized, it's just a de facto standard because PHP likes to do it this way.

So for now, your code in this PR loads all accessible days from all accessible camps. Maybe it would be easiest to add a `period.camp` filter to Day.php and use that instead? The frontend code would also get simpler then.

Reply:
done